iOS/Sample/Podfile.lock

```diff
@@ -0,0 +1,130 @@
+PODS:
```

Comment:
I agree Podfile.lock should be committed. I am working on it and trying to automatically update Podfile.lock when a release is made. Till then can you remove the podfile.lock and also add it in gitignore.